apps/web/app/meetings/_components/MeetingRoomSchedule.tsx

```diff
@@ -0,0 +1,31 @@
+"use client";
+
+import { type IReservation, type TBaseItem } from "@repo/types";
+import { useQuery } from "@tanstack/react-query";
+import { useDateStore } from "@/app/store/useDateStore";
+import { MEETING_ROOMS_TYPE } from "@/app/constants/meetingRoomsType";
+import { formatDate } from "@/app/utils/formatDate";
+import { getAllItems } from "@/api/items";
+import { getReservationsByTypeAndDate } from "@/api/reservations";
+import ScheduleTable from "./Schedule/ScheduleTable";
+import MeetingsSkeleton from "./skeleton";
+
+export default function MeetingRoomSchedule(): JSX.Element {
+  const { selectedDate } = useDateStore();
+
+  const formattedDate = formatDate(selectedDate);
+
+  const { data: meetingsData = [], isLoading: meetingsIsLoading } = useQuery<IReservation[]>({
+    queryKey: ["meetings", formattedDate, MEETING_ROOMS_TYPE],
```

Comment:
```suggestion
    queryKey: ["meetings", formattedDate, MEETING_ROOMS_TYPE],
```
queryKey를 적용할거면 "meetings" 부분도 상수화해서 주입해주는게 맞는거같아

Reply:
data: meetingsData = [] 부분은 isLoading으로 로딩처리를 했는데 빈 배열로 다시 초기화를 해줄 필요가 있을까?

Reply:
흠 생각해보니 그렇넹 그러면 로딩처리를 안 할 경우에는 배열로 초기화를 해야 할까? 굳이 안 해도 될 거 같기도 하구